fiftyone/utils/utils3d.py

```diff
@@ -539,39 +536,34 @@ def compute_orthographic_projection_images(
     if out_group_slice is not None:
         out_samples = []
 
-    all_metadata = []
-
-    with fou.ProgressBar(total=len(filepaths), progress=progress) as pb:
-        for filepath, group in pb(zip(filepaths, groups)):
-            image_path = filename_maker.get_output_path(
-                filepath, output_ext=".png"
-            )
+    for sample in view.iter_samples(autosave=True, progress=progress):
+        image_path = filename_maker.get_output_path(
+            sample.filepath, output_ext=".png"
+        )
 
-            img, metadata = compute_orthographic_projection_image(
-                filepath,
-                size,
-                shading_mode=shading_mode,
-                colormap=colormap,
-                subsampling_rate=subsampling_rate,
-                projection_normal=projection_normal,
-                bounds=bounds,
-            )
+        img, metadata = compute_orthographic_projection_image(
+            sample.filepath,
+            size,
+            shading_mode=shading_mode,
+            colormap=colormap,
+            subsampling_rate=subsampling_rate,
+            projection_normal=projection_normal,
+            bounds=bounds,
+        )
 
-            foui.write(img, image_path)
-            metadata.filepath = image_path
+        foui.write(img, image_path)
+        metadata.filepath = image_path
 
-            if out_group_slice is not None:
-                sample = Sample(filepath=image_path)
-                sample[group_field] = group.element(out_group_slice)
-                sample[metadata_field] = metadata
-                out_samples.append(sample)
+        sample[metadata_field] = metadata
 
-            all_metadata.append(metadata)
+        if out_group_slice is not None:
+            s = Sample(filepath=image_path)
+            s[group_field] = sample[group_field].element(out_group_slice)
+            s[metadata_field] = metadata
+            out_samples.append(s)
```

Comment:
@coderabbitai Using `autosave=True` is an acceptable tradeoff in this case because explicitly saving the sample inside a for loop can be very slow. Having the `autosave` option set causes save operations to be batched together for more optimal transmission to the database. If an error is encountered, it would be considered acceptable loss that previous updates are lost because they were in the batch queue waiting to be sent.